SIAB/data/interface.py

```
@@ -171,92 +118,136 @@ def radius_interpolation(element: str, radius_data: dict):
+        the name of unit output
+        
+    Returns
+    ------
+    float
+        converted data in unit specified by `to_`
+    '''
+    factors_ = {
+        'Ha': 1.0, 'a.u.': 1.0, 'au': 1.0, 'hartree': 1.0, 'Hartree': 1.0,
+        'Ry': 0.5, 'ry': 0.5, 'Rydberg': 0.5, 'rydberg': 0.5,
+        'eV': 27.21138602, 'ev': 27.21138602,
+        'kcal/mol': 627.509469, 'kcal': 627.509469, 'kcal/mol': 627.509469, 'kcal/mole': 627.509469,
```

Ruff

Dictionary key literal `'kcal/mol'` repeated

Remove repeated key literal `'kcal/mol'`

(F601)